packages/wallet-service/tests/api.test.ts

```diff
@@ -793,7 +798,8 @@ test('POST /wallet', async () => {
 
   // get the first address
   const xpubChangeDerivation = walletUtils.xpubDeriveChild(XPUBKEY, 0);
-  const firstAddress = walletUtils.getAddressAtIndex(xpubChangeDerivation, 0, process.env.NETWORK);
+  const firstAddressData = addressUtils.deriveAddressFromXPubP2PKH(xpubChangeDerivation, 0, process.env.NETWORK);
```

Comment:
Use the config to get the network?

Reply:
Did not seem necessary to refactor the tests to also use the config. 
What do you think?

Reply:
Maybe to guarantee that the config is being filled correctly?